environment/src/main/java/com/sequenceiq/environment/user/UserPreferencesService.java

```diff
@@ -0,0 +1,48 @@
+package com.sequenceiq.environment.user;
+
+import java.util.Optional;
+import java.util.UUID;
+
+import javax.inject.Inject;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.auth.security.authentication.AuthenticatedUserService;
+
+@Service
+public class UserPreferencesService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(UserPreferencesService.class);
+
+    @Inject
+    private UserPreferencesRepository userPreferencesRepository;
```

Comment:
Next time use constructor injection. I will fix it in my pull request.

Reply:
It is not a project policy.

Reply:
We talked about it, and we agreed to follow it in the environment service.